applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle DESKTOP_DVFS_STATE_$(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+if $(dvfs_timeout) != 0
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "timeout"
+	default $(dvfs_timeout)
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  DESKTOP_DVFS_STATE_$(dvfs_state) times out.
+	  A request is automatically cleared on timeout.
+
+endif # $(dvfs_state)_TIMEOUT_MS != 0
+
+if $(dvfs_timeout) = 0
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int
+	default 0
+	range 0 0
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  DESKTOP_DVFS_STATE_$(dvfs_state) times out.
+	  A request is automatically cleared on timeout.
+
+endif # $(dvfs_state)_TIMEOUT_MS = 0
+
+choice "DESKTOP_DVFS_STATE_$(dvfs_state)_ACTIVE_FREQ"
```

Comment:
It is a good practice to make the Kconfig choice options use the Kconfig choice name as a prefix:

DESKTOP_DVFS_STATE_$(dvfs_state)_ACTIVE_FREQ

DESKTOP_DVFS_STATE_$(dvfs_state)_ACTIVE_FREQ_HIGH
DESKTOP_DVFS_STATE_$(dvfs_state)_ACTIVE_FREQ_MEDLOW

OR

DESKTOP_DVFS_STATE_$(dvfs_state)_FREQ

DESKTOP_DVFS_STATE_$(dvfs_state)_FREQ_HIGH
DESKTOP_DVFS_STATE_$(dvfs_state)_FREQ_MEDLOW